lib/worker.js

```diff
@@ -143,17 +167,32 @@ Worker.prototype.run = function () {
         merge(jobAndQueueData, { result: result }),
         'Task complete'
       )
+      if (process.env.WORKER_MONITOR) {
+        var tags = this._eventTags()
+        tags.result = 'success'
+        monitor.increment('ponos.finish', tags)
+      }
```

Comment:
I agree, make this negative check, that way we do not have to add extra var in all our projects configs.